README.md

```diff
@@ -0,0 +1,72 @@
+# Nitrokey Python SDK
+
+The Nitrokey Python SDK can be used to list and manage Nitrokey devices.
```

Comment:
nit: "list" seems a bit weird as a feature to be mentioned. I would maybe say "use and configure Nitrokey devices"